docs/schedule/validate.zh_TW.md

```diff
@@ -0,0 +1,27 @@
+# GTFS Schedule驗證器
+
+這個免費和開源的規範GTFS Schedule驗證器由[MobilityData](https://mobilitydata.org)在GTFS社區。用它來根據官方評估你的數據集GTFS[參考](reference.md)和[最佳實踐](best-practices.md)。它以 HTML 格式提供易於使用的驗證報告，可以在瀏覽器中打開並與其他方共享。
+
+<!-- <img class="center" src="../../assets/validator_animation.gif" width="150"> -->
+
+<br/>
+
+<div class="usage-buttons"><a class="button" href="https://share.mobilitydata.org/validator-installer-mac">下載適用於 Mac<img class="icon" src="../../assets/apple.svg" width="20"/></a><a class="button" href="https://share.mobilitydata.org/validator-installer-windows">下載 Windows<img class="icon" src="../../assets/windows.svg" width="20"/></a><a class="button" href="https://share.mobilitydata.org/validator-installer-ubuntu">為 Ubuntu 下載<img class="icon" src="../../assets/ubuntu.svg" width="20"/></a></div>
+
+**版本 v4.0.0** / [閱讀發行說明](https://github.com/MobilityData/gtfs-validator/releases/latest)
+
+<hr/>
+
+## 用法
+
+<div class="usage"><div class="usage-list"><ol><li>下載基於我們操作系統的最新版本的項目。</li><li>安裝後，運行該應用程序，將出現一個窗口。</li><li>載入你的GTFS數據集：你可以指定一個url、ZIP 文件或包含提要的各個 .txt 文件的目錄。</li><li>選擇輸出目錄：這是將寫入驗證報告的目錄。</li><li>驗證完成後，結果將自動在您的瀏覽器中打開。如果看不到它們，只需打開在輸出路徑中創建的 report.html 文件。</li><li>您將看到驗證器是否發現了數據問題，並鏈接到我們的文檔以了解如何解決這些問題。驗證報告將保存為 HTML 文件，因此可以輕鬆保存並與他人共享。 </li></ol></div><div class="usage-video"><video class="center" width="560" height="315" controls=""><source src="../../assets/validator_demo_large.mp4" type="video/mp4"/></video></div></div>
+
+## 使用此驗證器作為數據管道的一部分
+
+要查看有關如何在數據管道中使用此工具的更多說明，請訪問[存儲庫](https://github.com/MobilityData/gtfs-validator)。
+
+## 貢獻
+
+這個項目是社區的努力，任何對這個項目感興趣的人都可以加入社區，參與決策並以不同的方式幫助推進它。有許多不同的貢獻方式：分享您對新功能的想法、報告問題、改進文檔、教其他人如何使用它等。
+
+有興趣為這個項目做貢獻嗎？請訪問[GitHub 存儲庫](https://github.com/MobilityData/gtfs-validator)並閱讀我們的[貢獻指南](https://github.com/MobilityData/gtfs-validator/blob/master/docs/CONTRIBUTING.md)。
```

Comment:
If I had to guess this is the right one because it has Github in the link, but what do I know, I am delirious after this review